feedback/static/html/scale_item.html

```diff
@@ -0,0 +1,13 @@
+<div class="feedback_likert_rating">
+  <label title="{scale_text}" class="feedback_likert_label">
```

Comment:
For accessibility, you have to group the radio buttons with the same name together. See
https://dequeuniversity.com/rules/axe/1.1/radiogroup

FYI @cptvitamin